midtransclient/config.py

```diff
@@ -21,7 +21,7 @@ def __init__(self,
         self.custom_headers = custom_headers
         self.proxies = proxies
 
-    def get_core_api_base_url(self):
+    def get_base_url(self):
```

Comment:
revert back the name to `get_core_api_base_url` because even the subscription & tokenization API should still be under core api